### PR TITLE
add vcr for tests

### PR DESCRIPTION
**Pull request by schmierkov**
@etagwerker @mauro-oto this adds vcr for the tests, check it out! thanks

### CODE REVIEW

**Comment by mauro-oto** [comment]
@schmierkov could you update this branch? There seems to be conflicts. Thanks!

**Comment by etagwerker** [comment]
@schmierkov Could you look into the failures in Travis?

**Comment by schmierkov** [comment]
@etagwerker had a look already, that's why I opened #30. I'm going to merge this PR to this branch here.

**Comment by schmierkov** [comment]
@etagwerker looks green now! 🎉

File: test/test_mercado_pago.rb [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
 require 'test/unit'
-require 'mercadopago'
+require 'vcr'
+
+#
+# Valid credentials to be used in the tests.
+#
+CREDENTIALS = {
+  client_id: '3962917649351233',
+  client_secret: 'rp7Ec38haoig7zWQXekXMiqA068eS376' }
```
Comment (etagwerker) [inline test/test_mercado_pago.rb]: @schmierkov Please remove real credentials from the test. We shouldn't depend on these being valid to test our code. If we make sure that these are not real, we can be sure that we are not really connecting to Mercado Pago.

**Comment by etagwerker** [comment]
@schmierkov Looks good, thanks! 🍻

**Pull request merged**